packages/nimbus/electron/preload.ts

```
@@ -23,10 +23,61 @@ contextBridge.exposeInMainWorld('ipcRenderer', {
+  },
+  
+  onAuthError: (callback: (error: string) => void) => {
+      const subscription = (_event: IpcRendererEvent, error: string) => callback(error);
+      ipcRenderer.on('auth:error', subscription);
+      return () => {
+          ipcRenderer.removeListener('auth:error', subscription);
+      };
+  }
+});
+
+export {};
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)